src/components/discussion/discussion.js

```diff
@@ -5,6 +5,7 @@ import PropTypes from 'prop-types';
 const Discussion = (props) => {
   return (
     <div className="discussion-box">
+      <h3>Giscus comments</h3>
```

Comment:
I think we need to indicate these are the new ones
```suggestion
      <h3>✨ Giscus comments</h3>
```